pkg/sql/logictest/testdata/logic_test/show_source

```diff
@@ -107,6 +107,15 @@ SELECT * FROM [SHOW KEYS FROM system.descriptor]
 Table       Name     Unique  Seq  Column  Direction  Storing  Implicit
 descriptor  primary  true    1    id      ASC        false    false
 
+query T colnames
+SELECT * FROM [SHOW SCHEMAS FROM system]
+----
+Schema
+crdb_internal
+information_schema
+pg_catalog
+public
```

Comment:
Can you please add a couple tests on user databases?

Reply:
but, but... system is a regular database, what does this buy us?

Reply:
anyway, test added

Reply:
Fair point... I guess it's just the principle. Also I see that you added a test for the form that doesn't use a `FROM` - thanks!